epimodel/exports/epidemics_org.py

```diff
@@ -117,7 +120,7 @@ def __init__(
         self.current_estimate = current_estimate
         # Any per-region data. Large ones should go to data_ext.
         self.data = self.extract_smallish_data(
-            rates, hopkins, foretold, timezones, un_age_dist, traces_v3
+            rates, hopkins, foretold, timezones, un_age_dist, traces_v3, countermeasures
         )
         # Extended data to be written in a separate per-region file
         self.data_ext = self.extract_external_data(
```

Comment:
Could you add it to the external data instead?

Reply:
I checked it and this doubles the main data size from 100kb -> 200kb so its actually fine. Lets do the splitting of in another PR where we also move the JH data to extfile.